Example/Auth/Tests/FIRUserTests.m

```diff
@@ -350,10 +352,61 @@ - (void)testUserProperties {
     #if TARGET_OS_IOS
     // Verify FIRUserInfo properties from the phone auth provider.
     id<FIRUserInfo> phoneUserInfo = providerMap[FIRPhoneAuthProviderID];
-    XCTAssertNotNil(phoneUserInfo);
     XCTAssertEqualObjects(phoneUserInfo.phoneNumber, kPhoneNumber);
     #endif
 
+    // Test NSSecureCoding
+    NSArray *paths =
+        NSSearchPathForDirectoriesInDomains(NSDocumentDirectory, NSUserDomainMask, YES);
+    NSString *documentsDirectory = [paths objectAtIndex:0];
+    NSString *filePath = [documentsDirectory stringByAppendingPathComponent:kFilePath];
```

Comment:
You don't need an actual file to test it. Use [initForWritingWithMutableData:](https://developer.apple.com/documentation/foundation/nskeyedarchiver/1409579-initforwritingwithmutabledata) so the tests run faster and doesn't depend on the file system.